source/jormungandr/jormungandr/interfaces/v1/fields.py

```diff
@@ -893,7 +898,8 @@ def output(self, key, obj):
 
 impacted_object = {
     'pt_object': NonNullNested(pt_object),
-    'impacted_stops': NonNullList(NonNullNested(impacted_stop))
+    'impacted_stops': NonNullList(NonNullNested(impacted_stop)),
+    'line_section': NonNullNested(impacted_line_section, attribute='impacted_line_sections') ## TODO to discuss during review, wouldn't 'impacted_section' be more consistent ?
```

Comment:
nop @xlqian there can be only one impacted section (on the impacted_object, but there can be several impacted_objects)